cognite/client/_api/assets.py

```diff
@@ -320,6 +321,47 @@ def aggregate(self, filter: Union[AssetFilter, Dict] = None) -> List[AssetAggreg
         """
         return self._aggregate(filter=filter, cls=AssetAggregate)
 
+    def aggregate_metadata_keys(self, filter: Union[AssetFilter, Dict] = None) -> Sequence[AggregateBucketResult]:
+        """`Aggregate assets <https://docs.cognite.com/api/v1/#operation/aggregateAssets>`_
+
+        Args:
+            filter (Union[AssetFilter, Dict]): Filter on assets filter with exact match
+
+        Returns:
+            Sequence[AggregateBucketResult]: List of asset aggregates
+
+        Examples:
+
+            Aggregate assets:
+
+                >>> from cognite.client import CogniteClient
+                >>> c = CogniteClient()
+                >>> aggregate_by_prefix = c.assets.aggregate_metadata_keys(filter={"external_id_prefix": "prefix"})
+        """
+        return self._aggregate(filter=filter, aggregate="metadataKeys", cls=AggregateBucketResult)
+
+    def aggregate_metadata_values(
+        self, keys: Sequence[str], filter: Union[AssetFilter, Dict] = None
+    ) -> Sequence[AggregateBucketResult]:
```

Comment:
Seems to return values case folded..? Could be quite surprising as the official documentation doesn't even mention it: https://docs.cognite.com/api/v1/#tag/Assets/operation/aggregateAssets

...however the docs under `events/aggregate` mention it explicitly:

_Also, in the case of text fields, the values are aggregated in a case-insensitive manner_

I think we should mention this in the docstring.

<img width="647" alt="Screenshot 2023-02-01 at 22 19 31" src="https://user-images.githubusercontent.com/8521241/216172211-6c8509c1-4232-47e4-8226-ea6f58a7c4ce.png">

Reply:
Sad, but not something we should fix in the SDK. I'll create a ticket for searchberg on this.

Reply:
(also a good catch 💯)

Reply:
And I'll update the docstring to mention this also 🤝